sdk/tests/js/webgl-test-utils.js

```diff
@@ -744,7 +744,8 @@ var glTypeToTypedArrayType = function(gl, type) {
     case gl.UNSIGNED_INT_2_10_10_10_REV:
     case gl.UNSIGNED_INT_24_8:
       return window.Uint32Array;
-    case gl.HALF_FLOAT:
+    case gl.HALF_FLOAT:      // 0x140B
+    case gl.HALF_FLOAT_OES:  // 0x8D61
```

Comment:
You should use 0x8D61 and put HALF_FLOAT_OES in the comment. HALF_FLOAT_OES is not part of WebGLRenderingContext.